src/gui/editorwidgets/qgsrangewidgetwrapper.cpp

```diff
@@ -142,10 +142,11 @@ void QgsRangeWidgetWrapper::initWidget( QWidget *editor )
     QgsSpinBox *qgsWidget = qobject_cast<QgsSpinBox *>( mIntSpinBox );
     if ( qgsWidget )
       qgsWidget->setShowClearButton( allowNull );
-    int minval = min.toInt();
+    int minval =  min.isValid() ? min.toInt() : std::numeric_limits<int>::lowest();
```

Comment:
It's okay IMO. But what is the reason doing it here and not do it on reading the config on line 90? https://github.com/qgis/QGIS/pull/32161/files#diff-76aae134610defb9f49b7af131458184R90